pages/profile/password/change.vue

```diff
@@ -193,3 +193,8 @@ const submitChangePassword = async () => {
     }
 }
 </script>
+<style>
+.fs-cp {
+    font-size: 2.5rem;
+}
```

Comment:
if you find that you have to make more font sizes, why not just extend bootstrap utility like we're already doing in variables.scss

Reply:
also use generic class names

Reply:
Done